spec/models/user_spec.rb

```diff
@@ -2374,4 +2374,8 @@ def filter_by(method)
       expect(user2.recent_time_read).to eq(50)
     end
   end
+
+  def reset_last_seen_cache!(user)
+    Discourse.redis.del("user:#{user.id}:#{Time.zone.now.to_date}")
+  end
```

Comment:
Minor but I prefer keeping this in the model proper, that way you get to reused the key calculation code that the model uses. 

I know it can sometimes feel a bit weird to have stuff like this in the model, so this is a bit of a bike shed. But overall this reduces code duplication which is a good thing.